app/controllers/users_controller.rb

```diff
@@ -38,6 +38,9 @@ def show
     @user = User.find(params[:id])
   end
 
+  def dashboard 
+    @user = User.find(params[:id])
```

Comment:
remove the @user object. you can use the current_user helper in your dashboard page